RFC-002-mapeo-protocol.md

```diff
@@ -173,6 +173,10 @@ of the form
 Each `name` must be unique, and should self-describe the file as well as
 possible.
 
+Not included (yet) are fields for valid existing Mapeo version ranges to
+upgrade from, which will be needed for incremental updates (JS bundles, or
+Electron `.blockmap` files).
```

Comment:
Just an FYI: The `.blockmap` files are gzipped JSON that list sizes and checksums for "bocks" within the `.exe`/`.dmg` file. They are used for downloading updates so that the client can only download ranges of the main installer file based on what ranges they already have from the previous installer. As such, the `.blockmap` file is not a partial update in itself, it is a way of a client choosing what ranges of an update file have changed so that it can only download those pieces. I don't _think_ it has applications in the p2p updates.

Reply:
Removed!